CMakeLists.txt

```
@@ -27,7 +27,21 @@ list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
 project( nest CXX C )
 set( NEST_USER_EMAIL "users@nest-simulator.org" )
 
-################################################################################
+# check if the given CMAKE_INSTALL_PREFIX is not empty
+if("${CMAKE_INSTALL_PREFIX}" STREQUAL "")
+  set(PathLength 0)
+  message(FATAL_ERROR "CMAKE_INSTALL_PREFIX cannot be an empty string" "CMAKE_INSTALL_PREFIX is:${CMAKE_INSTALL_PREFIX} [length:${PathLength}]")
```

```suggestion
  message(FATAL_ERROR "CMAKE_INSTALL_PREFIX cannot be set to an empty string")
```
I think the additional text is not that helpful, as the prefix is anyway empty and the length in this situation always 0.

---

I agree.